src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -736,7 +735,7 @@ def create_cc_resource(client, resource_group_name, cluster_name, cc, no_wait):
     try:
         return sdk_no_wait(no_wait, client.begin_create, resource_group_name=resource_group_name,
                            cluster_name=cluster_name, connected_cluster=cc)
-    except CloudError as e:
+    except Exception as e:
```

Comment:
`CloudError` has been replaced by `HttpResponseError` in the Python SDK of track 2. Why not use `HttpResponseError`? Is this because there are other exceptions that need to be caught?

Reply:
Yes, there could be other exceptions. We are catching it generically here and then handling the exception type in a common function 'arm_exception_handler'

Reply:
But just realized, i need to replace the CloudError from the 'arm_exception_handler' function and replace it with HttpResponseError

Reply:
Have replaced CloudError with HttpResponseError everywhere. Please review again